pkg/clients/dynatrace/settings_test.go

```diff
@@ -33,20 +85,19 @@ func TestDynatraceClient_GetMonitoredEntitiesForKubeSystemUUID(t *testing.T) {
 		require.NotNil(t, dtc)
 
 		// act
-		actual, err := dtc.GetMonitoredEntitiesForKubeSystemUUID(ctx, testUID)
+		actual, err := dtc.GetKubernetesClusterEntity(ctx, testUID)
```

Comment:
Should we also rename the ME reconciler as well?

Reply:
maybe, `k8sclusterentity.Reconciler` is a bit of a mouthful 😅

Reply:
renamed it to `k8sentity` added a doc for the package, also did bunch of renames everywhere related to it

cc @andriisoldatenko